.github/workflows/main.yml

```diff
@@ -139,6 +139,10 @@ jobs:
           os: macos-latest
           rust: nightly
           other: x86_64-apple-ios
+        - name: macOS aarch64 stable
+          os: macos-14
+          rust: stable
+          other: x86_64-apple-darwin
```

Comment:
Shouldn't this be `aarch64-apple-darwin`?

Reply:
Disregard this comment. 

`other` is for cross-compile tests, and or `arch64-apple-darwin` its `other` should be [`x86_64-apple-darwin`](https://github.com/rust-lang/cargo/blob/54281b804b4014e4238a227d65f5e8e14571912b/crates/cargo-test-support/src/cross_compile.rs#L212-L213) so this is correct.